docs/known_issues.md

```
@@ -0,0 +1,21 @@
+# Known Issues
+
+Issues have been reported with data, these vary by firmware versions. Newer firmware may solve these or introduce new issues.
+
+## Production & Consumption data
+
+| data                | envoy <br> type [^1] | issue                                    | in firmware |
+| ------------------- | -------------------- | ---------------------------------------- | ----------- |
+| whToday             | Mtr                  | late reset some time after midnight      |
```

markdownlint-cli2 (0.17.2)

Table column count
Expected: 4; Actual: 3; Too few cells, row will be missing data

(MD056, table-column-count)